packages/icons/src/svg/shine.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Look at the current icon patterns from the `svg` folder, every icon is using the `12px` version of the icon to be defined, could we update it to use the same?

Reply:
Done! Thanks for noticing 👍